contracts/RestrictedNORI.sol

```diff
@@ -880,9 +836,8 @@ contract RestrictedNORI is
     uint256 balanceOfAccount
   ) private view returns (uint256) {
     Schedule storage schedule = _scheduleIdToScheduleStruct[scheduleId];
-    uint256 scheduleTrueTotal = _scheduleTrueTotal(
-      schedule.totalClaimedAmount,
-      scheduleId
+    uint256 scheduleTrueTotal = schedule._scheduleTrueTotal(
```

Comment:
I think we should just be passing around `Schedule` as a parameter now so we don't keep re-loading it from the schedule id in the previous line

Reply:
good catch

Reply:
looked through the rest of this contract for any other places where something like this may be happening and I think we're in good shape.  found one small optimization to get rid of one more load